script.js

```diff
@@ -13,4 +12,210 @@ closeModalIcon.addEventListener('click', toggleMenuDisplay);
+popTitleDesk.className = 'pop-title-desk';
+popTitleClose.appendChild(popTitleDesk);
+popTitleDesk.textContent = projectData.desktopTitle;
+
+const popCloseImg = document.createElement('img');
+popCloseImg.src = projectData.imgTwo;
+popCloseImg.className = 'pop-close';
+popTitleClose.appendChild(popCloseImg);
+
+const popTechs = document.createElement('div');
+popTechs.className = 'pop-technologies';
+popContainer.appendChild(popTechs);
+
+const popTechList = document.createElement('ul');
+popTechList.className = 'pop-tech-list';
+popTechs.appendChild(popTechList);
+for (let i = 0; i < projectData.popTec.length; i += 1) {
+  const popList = document.createElement('li');
+  popList.innerText = projectData.popTec[i];
+  popTechList.appendChild(popList);
+}
+
+const popDescription = document.createElement('div');
+popDescription.className = 'pop-description';
+popContainer.appendChild(popDescription);
+
+const popImage = document.createElement('img');
+popImage.className = 'pop-image';
+popImage.src = projectData.popImg;
+popDescription.appendChild(popImage);
+
+const popParaImg = document.createElement('div');
+popParaImg.className = 'pop-para-img';
+popDescription.appendChild(popParaImg);
+
+const popParagraph = document.createElement('p');
+popParagraph.className = 'pop-paragraph';
+popParagraph.innerHTML = projectData.lorem;
+popParaImg.appendChild(popParagraph);
+
+/** ..Button.. */
+
+const popBtn = document.createElement('div');
+popBtn.className = 'pop-btn';
+
+const popBtnList = document.createElement('ul');
+popBtnList.className = 'pop-btn-list';
+popBtn.appendChild(popBtnList);
+popParaImg.appendChild(popBtn);
+
+const popBtnListItem = document.createElement('li');
+const popBtnListItem2 = document.createElement('li');
+popBtnListItem.className = 'btn-list-item';
+popBtnListItem2.className = 'btn-list-item';
+popBtnList.appendChild(popBtnListItem);
+popBtnList.appendChild(popBtnListItem2);
+
+const popButton = document.createElement('button');
+popButton.className = 'pop-button';
+const icn = '<i class="fa-solid fa-power-off"></i>';
+const val = `See Live ${icn}`;
+popButton.innerHTML = val;
+popBtnListItem.appendChild(popButton);
+
+const popButton2 = document.createElement('button');
+popButton2.className = 'pop-button';
+const icn2 = '<i class="fa-brands fa-github"></i>';
+const val2 = `See Source ${icn2}`;
+popButton2.innerHTML = val2;
+popBtnListItem2.appendChild(popButton2);
+
+const buttonPop = document.querySelector('.popup-btn');
+let bol = false;
+buttonPop.addEventListener('click', () => {
+  const flowBig = popContainerBig.style;
+  flowBig.display = 'block';
+  bol = true;
+  if (bol === true) {
+    bodyMain.style.overflow = 'hidden';
+  }
+});
+
+const popClose = document.querySelector('.pop-close');
+popClose.addEventListener('click', () => {
+  const popContBig = popContainerBig.style;
+  popContBig.display = 'none';
+  if (popContBig.display === 'none') {
+    bodyMain.style.overflow = 'auto';
+  }
+});
```

Comment:
- [ ] Kindly make sure that the modal popup works for each project card currently, only the first project modal card works properly, please make sure the other project  cards have modals popup


https://user-images.githubusercontent.com/69638013/216056742-61b6cf91-c609-48cd-926d-0f0da1a26dd1.mp4

Reply:
Fixed.